app/Http/Controllers/OutgoingLetterLogsController.php

```diff
@@ -12,15 +12,11 @@ class OutgoingLetterLogsController extends Controller
 {
     public function index(Request $request)
     {
+        $after_date = $request->input('after') ?? OutgoingLetterLog::min('date');
+        $before_date = $request->input('before') ?? OutgoingLetterLog::max('date');
```

Comment:
you can directly use `$request->after` and `$request->before` instead of calling `input` method on request.